### PR TITLE
Fixed: safer raws checks.

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

https://github.com/stylelint/stylelint/issues/2144 and future issue with potential undefined properties in `raws` object.

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

I hope nothing is missed :smile: 

/сс @davidtheclark @jeddy3

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Why we don't use `lint` script before test?

**Comment by davidtheclark** [comment]
> before the send PR I can potentially have style code issue

If you'd like to make a single npm script that runs linting and also runs all the tests (both tape and jest), feel free.

**Comment by davidtheclark** [comment]
👍  thanks @evilebottnawi!

**Comment by evilebottnawi** [comment]
@davidtheclark It should be something to add to `CHANGELOG.md`?

**Comment by davidtheclark** [comment]
@evilebottnawi I don't know what to add. The issue was too long and indirect. Can you summarize?

**Comment by evilebottnawi** [comment]
@davidtheclark nope, just asked :smile:

**Comment by davidtheclark** [comment]
@evilebottnawi Thanks for helping me to remember!

**Comment by ntwb** [comment]
For reference, `lint` is now ran as part of `npm test` via #2157
